### ISSUE TITLE
Accessing missing method (ApplicationVariant.getMappingFileProvider()) in version 2.4.0 

### ISSUE THREAD

**Issue by dimipaun** [Triple-T/gradle-play-publisher#693]
### Describe the bug

Build fails with release 2.4.0  and gradle 2.6.2: 

```
* Exception is:
java.lang.NoSuchMethodError: com.android.build.gradle.api.ApplicationVariant.getMappingFileProvider()Lorg/gradle/api/provider/Provider;
        at com.github.triplet.gradle.play.tasks.internal.PublishArtifactTaskBase.getMappingFile$plugin(PublishArtifactTaskBase.kt:47)
        at org.gradle.api.internal.tasks.properties.bean.AbstractNestedRuntimeBeanNode$BeanPropertyValue$1$1.create(AbstractNestedRuntimeBeanNode.java:80)
        at org.gradle.util.SingleMessageLogger.whileDisabled(SingleMessageLogger.java:467)
        at org.gradle.api.internal.tasks.properties.bean.AbstractNestedRuntimeBeanNode$BeanPropertyValue$1.get(AbstractNestedRuntimeBeanNode.java:76)
        at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:167)
        at org.gradle.api.internal.tasks.properties.bean.AbstractNestedRuntimeBeanNode$BeanPropertyValue.call(AbstractNestedRuntimeBeanNode.java:148)
        at org.gradle.util.GUtil.uncheckedCall(GUtil.java:461)
```

#### Tasks executed

What tasks did you run? For example, `publishBackstageReleaseBundle`.

**Comment by jeffdcamp** [comment]
I'm seeing the same issue

**Comment by SUPERCILEX** [comment]
Oops, my B. Fixed in [v2.4.1](https://github.com/Triple-T/gradle-play-publisher/releases/tag/2.4.1) by faa3d0ea996932f021ae44ac073612f7541ababb.
